cdk/lib/cdk-stack.ts

```diff
@@ -3,17 +3,43 @@ import { Construct } from "constructs";
 
 import * as s3 from "aws-cdk-lib/aws-s3";
 import * as s3deploy from "aws-cdk-lib/aws-s3-deployment";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+
+const BUILD_DIR = "../build";
 
 export class CdkStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props?: cdk.StackProps) {
-    super(scope, id, props);
+    super(scope, id, {
+      ...props,
+      analyticsReporting: false, // Disable CDK Metadata
+    });
+
+    const subwayPageLambda = new lambda.Function(this, "SubwayPageLambda", {
+      runtime: lambda.Runtime.PROVIDED_AL2023,
+      code: lambda.Code.fromAsset(`${BUILD_DIR}/bundles/home.zip`),
+      handler: "handler",
```

Comment:
I am not sure if these are all the parameters I need here, because I did a cloudformation import via the UI using the `cdk` generated templates.

Either way, we're missing all the API Gateway and DNS stuff we have going on in this CDK so all this is doing for us is making uploading changed assets simpler (previously I was going through the console and uploading them from my local disk, which was _horrible_)